05/app.js

```diff
@@ -1,3 +1,42 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+
+const deleteBtn = curr.cloneNode(false);
+//console.dir(deleteBtn.classList);   //?dev to z tym consol.dir nie działa, dale jest to wyświetlane jako referencja zeminna dynamicznie. Jest jakiś inny sposób? Jakaś inna medota na objecie consol.???
```

Comment:
Dotyczyło to elementów drzewa DOM, tutaj trzeba utworzyć kopie obiektu np. `{ ...deleteBtn.classList }`

Reply:
Ja chyba nie do końca to kumam, ale doczytam sobie o tych operatorach rest i spread i jeszcze raz tu zerknę 

- [ ] - a odhaczę jak zerknę ;P